project/SemanticConventionsGenerator.scala

```diff
@@ -0,0 +1,70 @@
+import sbt._
+import scala.sys.process._
+
+object SemanticConventionsGenerator {
+
+  private val generatorVersion = "0.18.0"
+
+  // generates semantic conventions by using `otel/semconvgen` in docker
+  def generate(version: String, rootDir: File): Unit = {
+    val semanticConventionsRepoZip =
+      s"https://github.com/open-telemetry/semantic-conventions/archive/v$version.zip"
+
+    val schemaUrl =
+      s"https://opentelemetry.io/schemas/$version"
+
+    val buildDir = rootDir / "buildscripts" / "semantic-convention"
+    val zip = buildDir / "semantic-conventions.zip"
+    val conventionsDir = buildDir / s"semantic-conventions-$version"
+
+    IO.delete(zip)
+    IO.delete(conventionsDir)
+
+    (url(semanticConventionsRepoZip) #> zip).!
+
+    IO.unzip(zip, buildDir)
+
+    // format: off
+    val genAttributes = List(
+      "docker",
+      "run",
+      "--rm",
```

Comment:
Hmm. Is Scala Steward able to run docker images? 🤔 I think it may be running in a docker itself.

Reply:
Ah, I forgot Scala Steward runs in docker.  Then it won't work. :(

Reply:
Actually, I might be wrong about this. Maybe the public steward is running in docker, but I think the Typelevel steward is just running in ordinary GHA.

https://github.com/scala-steward-org/scala-steward-action/blob/a80ff5bf69947d9be97d9eda20d5e1eb9faba209/src/action/main.ts#L54

Reply:
🤔 let's give it a try. 

Currently, there is only one release in the https://github.com/open-telemetry/semantic-conventions-java. 

A new release of the [Semantic Conventions](https://github.com/open-telemetry/semantic-conventions) should be published soon https://github.com/open-telemetry/semantic-conventions/pull/380. Java release will follow eventually. 

Then, Scala Steward will try to do its work.